src/Microsoft.Graph.Core/Extensions/HttpClientExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph
+{
+    using System.Collections.Generic;
+    using System.Net.Http;
+
+    internal static class HttpClientExtensions
+    {
+        internal static void SetFeatureFlags(this HttpClient httpClient, IEnumerable<string> flags)
```

Comment:
We'll need to convert the featureFlags enum to a string before adding the header.